packages/provider/src/MultiChainProvider.ts

```diff
@@ -112,26 +131,33 @@ export class MultiChainProvider extends SafeEventEmitter implements Provider {
     return response.result;
   }
 
+  /**
+   * Get the provider that is injected by the wallet, i.e., `window.ethereum`.
+   *
+   * @returns The injected provider.
+   */
   #getProvider() {
-    return window.ethereum;
+    return ethereum;
```

Comment:
Is this safe 🤔

Reply:
I guess so? It just defaults to `global` on Node.js, and `window` in browsers.

Reply:
Could we use `globalThis`?

Reply:
May need `declare global`

Reply:
```ts
declare global {
  const ethereum: SnapProvider;
}

globalThis.ethereum;
// TS2339: Property 'ethereum' does not exist on type 'typeof globalThis'.
```